.ci/Jenkinsfile

```
@@ -126,6 +136,10 @@ pipeline {
       options {
         warnError('Pull Requests to APM agents failed')
       }
+      environment {
+        // agentMapping is defined in the shared library as a map
+        SELECTED_AGENT = agentMapping.id(params.APM_AGENTS)
```

Or you could set the APM_AGENTS in the top level environment closure

```suggestion
        APM_AGENTS = "${params?.APM_AGENTS}"
        SELECTED_AGENT = agentMapping.id(env.APM_AGENTS)
```